src/status_im2/subs/chat/messages.cljs

```diff
@@ -146,22 +146,9 @@
 
 (re-frame/reg-sub
  :chats/pinned
- (fn [[_ chat-id] _]
```

Comment:
Don't rely on a subscription on `messages` in the database, as those are only the ones available to the user in the chat view (last 20 by default).